### PR TITLE
Removes the extra disks from vSphere environment

### PR DESCRIPTION
**Pull request by vavuthu**
Description: Currently when we teardown the cluster, it
leaves the disks in datastore. This fix will delete the
disk from datastore.

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: ocs_ci/deployment/vmware.py [hunk 1]
```diff
@@ -236,6 +236,33 @@ def modify_scaleup_repo(self):
...
+        vms = self.get_compute_vms(self.datacenter, self.cluster)
+        for vm in vms:
+            self.vsphere.remove_disks(vm)
+
+    def get_compute_vms(self, dc, cluster):
+        """
+        Gets the compute VM's from resource pool
+
+        Args:
+            dc ( str): Datacenter name
```
Comment (petr-balogh) [inline ocs_ci/deployment/vmware.py]: extra space before str

File: ocs_ci/utility/vsphere.py [hunk 2]
```diff
@@ -467,3 +467,74 @@ def destroy_pool(self, pool, dc, cluster):
...
+        disk_label = f"{disk_prefix}{unit_number}"
+
+        virtual_disk_device = None
+        for dev in vm.config.hardware.device:
+            # choose the device based on unit number instead of
+            # deviceInfo.label. labels can change if a disk is removed
+            if (
+                    isinstance(dev, vim.vm.device.VirtualDisk)
+                    and dev.unitNumber == unit_number
+                    and disk_prefix in dev.deviceInfo.label
```
Comment (petr-balogh) [inline ocs_ci/utility/vsphere.py]: One indent left

Comment (vavuthu) [reply ocs_ci/utility/vsphere.py]: Done

File: ocs_ci/deployment/vmware.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/deployment/vmware.py]: Done

**Pull request merged**